script/infoRelease.sh

```diff
@@ -84,27 +83,37 @@ Target Product Build: $targetProdBuild
 Source Product Tag: $productTag
 Community Tag (if available): $communityTag
  
+Product Pages: 
+https://pp.engineering.redhat.com/pp/product/jbossbrms/overview
+https://pp.engineering.redhat.com/pp/product/jbossbpms/overview
 
```

Comment:
Lets keep this hardcoded for now... we can externalize this into a properties file in the future if necessary.